src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/EventServiceImpl.java

```diff
@@ -227,12 +227,11 @@ private void fireEventChangeHandlers(final MessageType messageType, final Event
               try {
                 handler.handleEventLifecycle(messageType, event);
               } catch (CTPException e) {
-                log.error(
-                    "Failed to handle event change for {} of {} - {} ({})",
-                    messageType,
-                    event.getId(),
-                    e.getMessage(),
-                    e.getFault());
+                log.with("message_type", messageType)
```

Comment:
when we're logging an exception, we should put the exception as the final parameter in the method call to either `.warn` or `.error` and let the logging framework log the cause and the stack trace etc.